libs/react-components/src/tedi/components/cards/card/card-header/card-header.tsx

```diff
@@ -0,0 +1,97 @@
+import cn from 'classnames';
+import React from 'react';
+
+import { BreakpointSupport, useBreakpointProps } from '../../../../helpers';
+import styles from '../card.module.scss';
+import CardContent, { CardContentPadding, CardContentProps } from '../card-content/card-content';
+import { CardContext } from '../card-context';
+import { CardBackground, getPaddingCssVariables } from '../utility';
+
+export type CardHeaderBreakpointProps = {
```

Comment:
When it's duplicated props from CardContent and we render inside a CardContent, why we cant reuse the props?

When we want to still export CardHeaderProps separately, we can sue something like:
`export type CardHeaderProps = CardContentProps` (but i dont think it's really necessary)